src/nodes/index.js

```diff
@@ -16,12 +16,12 @@ module.exports = {
   Block: require('./Block.js'),
   BooleanLiteral: require('./BooleanLiteral.js'),
   BreakStatement: require('./BreakStatement.js'),
+  CatchClause: require('./CatchClause.js'),
```

Comment:
this should be automatically generated by running the `generateindexes` script

Reply:
I ran this but the file didn't change except for the timestamp, so I guess it doesn't make sense to push that.

Reply:
no I mean you added the new parts by hand while instead you should have just run the script or am I missing something?

Reply:
thanks, now we know this has been generated automatically 👍